app/move/controllers/create/save.js

```diff
@@ -23,13 +24,26 @@ class SaveController extends CreateBaseController {
     }
   }
 
-  successHandler(req, res) {
-    const { id } = req.sessionModel.get('move')
+  async successHandler(req, res, next) {
+    const { id, from_location: fromLocation } = req.sessionModel.get('move')
+    const journeyDuration = Math.round(
+      new Date().getTime() - req.sessionModel.get('journeyTimestamp')
+    )
 
-    req.journeyModel.reset()
-    req.sessionModel.reset()
-
-    res.redirect(`/move/${id}/confirmation`)
+    try {
+      await analytics.sendJourneyTime({
+        utv: capitalize(req.form.options.name),
```

Comment:
I understand we call that `name` already, but we mean something like `journeyName`. Maybe we could think of renaming it at some point to make it clearer (after all, we use names for people and for fields as well, and several other things).

Reply:
Oops, scratch that -- it comes from HOF, doesn't it.

Reply:
Yep, unfortunately it's a HOF thing :(